Makefile.am

```diff
@@ -185,7 +185,7 @@ motr_libmotr_la_CPPFLAGS  = -DM0_TARGET='libmotr' $(AM_CPPFLAGS)
 motr_libmotr_la_LDFLAGS   = -version-info @LT_VERSION@ -pthread $(AM_LDFLAGS)
 motr_libmotr_la_LIBADD    = @MATH_LIBS@ @PTHREAD_LIBS@ @AIO_LIBS@ @RT_LIBS@ \
                             @YAML_LIBS@ @PROFILER_LIBS@ @UUID_LIBS@ @GALOIS_LIBS@ \
-                            @DL_LIBS@ @CASSANDRA_LIBS@ @UV_LIBS@
+                            @DL_LIBS@ @CASSANDRA_LIBS@ @UV_LIBS@ @LIB_FAB@
```

Comment:
Naming.
All other libraries are called "XXX_LIBS".
I think for this patch, we need to call it "LIBFAB_LIBS"?